.github/workflows/tests.yml

```diff
@@ -118,3 +118,22 @@ jobs:
         # don't check /etc/os-release sourcing, allow useless cats to live inside our codebase, and
         # allow seemingly unreachable commands
         SHELLCHECK_OPTS: -e SC1091 -e SC2002 -e SC2317
+
+  python-lint:
+    name: "🐍 Lint (test scripts)"
+    runs-on: ubuntu-latest
+    container:
+      image: registry.fedoraproject.org/fedora:39
```

Comment:
You could probably use the `latest` tag to get the latest supported Fedora release. We would not need to think about updating this line in the future. Although it may also introduce unwanted surprises when new Fedora version is GA...

Reply:
I had the same thought and went with fixed version but I think I'm changing my mind now.  Maybe it's okay if we get a surprise failure when the version is updated :shrug: